src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -281,3 +288,31 @@ public partial struct JsonWriterState
         public void WriteStringValue(string value, bool escape = true) { }
     }
 }
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonSerializer
+    {
+        public static object Parse(System.ReadOnlySpan<byte> utf8Json, System.Type returnType, System.Text.Json.Serialization.JsonSerializerOptions options = null) { throw null; }
+        public static object Parse(string json, System.Type returnType, System.Text.Json.Serialization.JsonSerializerOptions options = null) { throw null; }
+        public static TValue Parse<TValue>(System.ReadOnlySpan<byte> utf8Json, System.Text.Json.Serialization.JsonSerializerOptions options = null) { throw null; }
+        public static TValue Parse<TValue>(string json, System.Text.Json.Serialization.JsonSerializerOptions options = null) { throw null; }
+        public static System.Threading.Tasks.ValueTask<object> ReadAsync(System.IO.Stream utf8Json, System.Type returnType, System.Text.Json.Serialization.JsonSerializerOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static System.Threading.Tasks.ValueTask<TValue> ReadAsync<TValue>(System.IO.Stream utf8Json, System.Text.Json.Serialization.JsonSerializerOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static byte[] ToBytes(object value, System.Type type, System.Text.Json.Serialization.JsonSerializerOptions options = null) { throw null; }
```

Comment:
It was unresolved during the review. I have a "todo" in the declaration. I can't remove it because we need some easy way to get the raw bytes out for practicality.

There was also a concern about the naming - we didn't want to "use up" a "ToJson" or similar if we add a utf8Char type. I think "ToUtf8Bytes" would work however although it seems somewhat unnecessary to me.

And yes we will probably want an overload to copy into a user-specified buffer\memory - I'll bring this up in the next review.